h2o-py/tests/testdir_generic_model/pyunit_generic_model_mojo_model_gbm.py

```diff
@@ -0,0 +1,49 @@
+import h2o
+import tempfile
+import os
+from h2o.estimators import H2OGradientBoostingEstimator, H2OGenericEstimator
+from tests import pyunit_utils
+
+
+def mojo_model_test():
+
+    # GBM
+    airlines = h2o.import_file(path=pyunit_utils.locate("smalldata/testng/airlines_train.csv"))
+    gbm = H2OGradientBoostingEstimator(ntrees = 1)
+    gbm.train(x = ["Origin", "Dest"], y = "IsDepDelayed", training_frame=airlines)
+
+    original_model_filename = tempfile.mkdtemp()
+    original_model_filename = gbm.download_mojo(original_model_filename)
+    
+    key = h2o.lazy_import(original_model_filename)
+    fr = h2o.get_frame(key[0])
+    model = H2OGenericEstimator(mojo_key = fr)
+    model.train()
+    predictions = model.predict(airlines)
+    assert predictions is not None
+    assert predictions.nrows == 24421
+    assert model._model_json["output"]["variable_importances"] is not None
+    assert len(model._model_json["output"]["variable_importances"]._cell_values) > 0
+    assert model._model_json["output"]["model_summary"] is not None
+    assert len(model._model_json["output"]["model_summary"]._cell_values) > 0
+    
+    # Test constructor generating the model from existing MOJO file
+    model = H2OGenericEstimator.from_mojo_file(original_model_filename)
+    assert model is not None
+    predictions = model.predict(airlines)
+    assert predictions is not None
```

Comment:
Do you want to also compare the model prediction and mojo prediction contents as well to make sure they are equal.  Perhaps just for a few row to cut down test run time?

Reply:
This is actually done in GenericModelTest.java. the R&Python client tests only make sure the client actually integrates well with H2O backend. The integration pipeline is well tested elsewhere, so this can be safely tested just on Backend.